README.md

```diff
@@ -432,7 +438,11 @@ The `draft-release` action is used to augment release notes generated by the `re
 
 The `draft-release` action needs to be able to look up and remotely fetch buildpack metadata. It does this by obtaining the `buildpack.toml` file for any dependent buildpacks from Github. This requires mapping the image URI to the Github project URI, which is not always one-to-one. As such, the action supports mappers.
 
-A mapper will add additional permutations of Github project URIs to try. For example, if you have an image URI of `gcr.io/foo/bar:1.2.3` but the Github project is at `github.com/org/foo-bar` then you can add a mapper of `|foo\/bar|org/foo-bar|` to have `draft-release` try both. You can specify as many mappers as you want, with each mapper generating a new URI for `draft-release` to try. It will stop when it hits the first URL that successfully pulls back the `buildpack.toml` file.
+A mapper will add additional permutations of Github project URIs to try. For example, if you have an image URI of `gcr.io/foo/bar:1.2.3` but the Github project is at `github.com/org/foo-bar` then you can add a mapper of `|foo\/bar|org/foo-bar|` to have `draft-release` try both. You can specify as many mappers as you want, with each mapper generating a new URI for `draft-release` to try. It will stop when it hits the first URL that successfully pulls back the `buildpack.toml` file. Mappers may use regexp capture groups and reference the captured value with `$<num>` (ex: `$1`, `$2`, etc) in the regular expression.
+
+The location of the `buildpack.toml` file in the Github project is assumed to be at `/buildpack.toml`. If you have a monorepo with multiple buildpacks, you may adjust the location by setting an environment variable for the action in the format `BP_TOML_PATH_<org>_<repo>=actual/path` where `<org>` and `<repo>` are the organization and repo pulled out of the image URI. For example, `gcr.io/foo/bar:1.2.3`. The org is `foo` and the repo is `bar`. 
+
+A leading `/` on the environment variable is not necessary, nor is `buildpack.toml` on the end. You may omit both to shorten the value and the action adjust as required. For example, with an org of `tanzu-buildpacks` and a repo of `java-function` then the environment variable would be `BP_TOML_PATH_TANZU_BUILDPACKS_JAVA_FUNCTION='/buildpacks/java/buildpack.toml`, which would be the same as `BP_TOML_PATH_VMWARE_TANZU_FUNCTION_BUILDPACKS_FOR_KNATIVE='buildpacks/java` since the trailing `buildpack.toml` is optional.
```

Comment:
Should this be:
```suggestion
A leading `/` on the environment variable is not necessary, nor is `buildpack.toml` on the end. You may omit both to shorten the value and the action adjust as required. For example, with an org of `tanzu-buildpacks` and a repo of `java-function` then the environment variable would be `BP_TOML_PATH_TANZU_BUILDPACKS_JAVA_FUNCTION=/buildpacks/java/buildpack.toml`, which would be the same as `BP_TOML_PATH_TANZU_BUILDPACKS_JAVA_FUNCTION=buildpacks/java` since the trailing `buildpack.toml` is optional.
```